setup/openchange_provision

```diff
@@ -29,6 +29,24 @@ import samba
 import samba.getopt as options
 import openchange.provision as openchange
 
+def check_incompatible_options(parser, opts):
```

Comment:
They dont add anything there. In fact they would be more awkward since I will have to pass to the callback information about the values of other options